backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/invoicingpool/DocTypeInvoicingPool.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2023 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.document.invoicingpool;
+
+import de.metas.document.DocTypeId;
+import de.metas.money.Money;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+@Builder
+public class DocTypeInvoicingPool
+{
+	@NonNull
+	String name;
+
+	@NonNull
+	DocTypeId positiveAmountDocTypeId;
+
+	@NonNull
+	DocTypeId negativeAmountDocTypeId;
+
+	@NonNull
+	Boolean isSoTrx;
+	
+	@NonNull
+	Boolean isActive;
```

Comment:
why not `boolean` primitive?

Reply:
to make sure we get an error if it's omitted in the builder

Reply:
@cp-ps  then pls create a ctor and have it there as Boolean.
Even though i don't see it necessary.

You could do
`@Default boolean isActive = true`
and i guess u would be on the safe side

wdyt?

But the actual model shall be "clean".

Reply:
saw that u changed for DocTypeInvoicingPoolCreateRequest but not here. any reasons why?

Reply:
Sorry, I hadn't seen your second comment only after I committed the first requested changes.
In the end, I used `@Default boolean isActive = true`.